README.md

```diff
@@ -12,7 +12,7 @@ To get a quick overview of the config file, see an example file [here](https://g
 
 ## Model Specification
 
-A model entry in the bioimage.io model zoo is defined by a configuration file `<model name>.model.yaml`.
+A model entry in the bioimage.io model zoo is defined by a configuration file `model.yaml`.
```

Comment:
```suggestion
A model entry in the bioimage.io model zoo is defined by a configuration file `<optional model name>.model.yaml`. Once a model is packaged to a `<(mandatory) model name>.model.bioimage.io.zip` file the configuration file inside the .zip is only named `model.yaml`.
```
The model entry is not the same as the packaged model (where we agreed to `model.yaml` inside a `<model name>.model.bioimage.io.zip`, see https://github.com/bioimage-io/configuration/issues/41). Keeping several models in one folder should be possible, as the may share other resources, etc.

Reply:
On second thought, I am not sure we should mention this here, it's difficult to understand this without explaining the `...model.zip` (which we are not doing yet, but need to add here as well.)